clever.go

```diff
@@ -225,6 +225,19 @@ type Section struct {
 	Term
 }
 
+type EventResp struct {
+	Links []Link
+	Event Event `json:"data"`
+	Uri   string
+}
+
+type Event struct {
+	Type    string
+	Created string
+	Id      string
+	Data    map[string]interface{}
```

Comment:
What are your thoughts on having the `object` field as well? So something like this:

```go
type Event struct {
  Type string
  Created string
  Id string
  Data struct {
    Object map[string]interface{}
  }
}
```